packages/components/src/components/checkbox-wrapper/checkbox-wrapper-styles.ts

```diff
@@ -32,13 +44,39 @@ export const getComponentCss = (
               `<path fill="${indeterminateIconColor}" d="m20,11v2H4v-2h16Z"/>`
             ),
           },
-          ...(!isDisabled && {
-            '&(input:focus)::before': {
-              borderRadius: borderRadiusMedium,
-            },
-          }),
+          ...(!isDisabled &&
+            !isLoading && {
+              '&(input:focus)::before': {
+                borderRadius: borderRadiusMedium,
+              },
+            }),
         }),
       },
+
+      spinner: {
```

Comment:
Can't we make this conditionally? Currently it bloats up the DOM especially in SSR Context for a less common usecase.

Reply:
@denyo see comment https://github.com/porsche-design-system/porsche-design-system/pull/2483#discussion_r1166868749
If making it conditional depending on "loading" state, css transition won't work.

But we can exclude this rule for ssr.

Reply:
Oli fixed it: because of accessibility, it makes sense to render spinner conditionally